composer.json

```diff
@@ -78,7 +78,8 @@
         "bower-asset/jquery": "2.2.*@stable | 2.1.*@stable | 1.11.*@stable | 1.12.*@stable",
         "bower-asset/jquery.inputmask": "~3.2.2 | ~3.3.3",
         "bower-asset/punycode": "1.3.*",
-        "bower-asset/yii2-pjax": "~2.0.1"
+        "bower-asset/yii2-pjax": "~2.0.1",
+        "psr/http-message": "^1.0"
```

Comment:
as far as I see this would be an optional dependency which could be in require-dev, we only test for an interface, this does not need to exist so not everyone installing yii needs to install `psr/http-message`.